apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -127,6 +127,10 @@ type EnvoySettings struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// PodAnnotations holds the annotations that will be add to the envoy‘s pod.
```

Comment:
Little nit on wording (will need to re-run `make generate` after this as well):
```suggestion
	// PodAnnotations defines annotations to add to the Envoy pods.
```

Reply:
done